README.md

```diff
@@ -35,6 +35,11 @@ Running using ~/.my.cnf:
 
     ./mysqld_exporter <flags>
 
+Format for flags (example):
+  
+    -collect.auto_increment.columns
```

Comment:
We have switched to kingpin for flags, so these have a different format now.

Boolean flags now take the form `--foo` and `--no-foo`.